Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -446,14 +447,37 @@ bool EmbeddedFrag::matchToTemplate(const RDKit::INT_VECT &ringSystemAtoms,
        coordinate_templates.getMatchingTemplates(ringSystemAtoms.size())) {
     // To reduce how often we have to do substructure matches, check ring info
     // and bond count first
-    if (mol->getNumBonds() != rs_mol.getNumBonds()) {
+    if (mol->getNumBonds() != numBonds) {
       continue;
     } else if (mol->getRingInfo()->numRings() != ring_count) {
       continue;
     }
+    // also check if the mol atoms have the same connectivity as the template
+    auto degreeCounts = [](const RDKit::ROMol &mol) {
```

Comment:
This small change is required for Windows builds:
```suggestion
    auto degreeCounts = [DUMMY_ATOMIC_NUM](const RDKit::ROMol &mol) {
```